### DISCUSSION TITLE
ketcher-core documentation

### DISCUSSION THREAD

**Discussion by QDMarkMan** [epam/ketcher#739] [Q&A] [answered]
I want to develop a vue component based on ketcher-core.  Do you have documentation for ketcher-core ?   If not, may I ask if you will write documentation for ketcher-core in the future ?

**Comment by AndreiMazol** [comment, marked as answer]
Hi @QDMarkMan ,

The documentation will be provided as a part of release 2.4. Planned release date is 15th October.

Best regards,
Andrei

**Reply by QDMarkMan** [reply]
Thank you very much for your reply！
